integreat_cms/static/src/css/style.scss

```
@@ -227,14 +227,22 @@ select,
 [type="radio"] {
     @apply rounded text-blue-500 mr-2 align-text-top;
 
-    &:disabled {
+    &:disabled,
+    .fake-disable {
         @apply bg-gray-200 border-gray-200;
```

```suggestion
        @apply bg-gray-200 border-gray-200 pointer-events-none;
```
And this class can be applied already here, the we don't have to do it in js.